src/actions/clusterActions.js

```diff
@@ -553,12 +547,12 @@ function clusterLoadStatusV4(dispatch, clusterId) {
  *
  * @param {Object} cluster Cluster definition object
  * @param {Boolean} isV5Cluster
+ * @param {Array} nodePools array of node pool objects
  */
 export function clusterCreate(cluster, isV5Cluster) {
   return function(dispatch) {
```

Comment:
We could make this an arrow function

Reply:
Totally. 
I won't do it because this is the pattern I found and we have followed in all thunks.
I think it is interesting to follow the same patterns for thunks, just for readibility sake.